examples/CMakeLists.txt

```
@@ -38,12 +38,12 @@ dd4hep_configure_output()
 
 #==========================================================================
 
-SET(DD4HEP_BUILD_EXAMPLES "AlignDet CLICSiD ClientTests Conditions DDCMS DDCodex DDDB DDDigi DDG4 DDG4_MySensDet LHeD OpticalSurfaces Persistency SimpleDetector"
+SET(DD4HEP_EXAMPLES "AlignDet CLICSiD ClientTests Conditions DDCMS DDCodex DDDB DDDigi DDG4 DDG4_MySensDet LHeD OpticalSurfaces Persistency SimpleDetector"
   CACHE STRING "List of DD4hep Examples to build")
-SEPARATE_ARGUMENTS(DD4HEP_BUILD_EXAMPLES)
+SEPARATE_ARGUMENTS(DD4HEP_EXAMPLES)
 MESSAGE(STATUS "Will be building these examples: ${DD4HEP_BUILD_EXAMPLES}")
```

You have to change also this
```suggestion
MESSAGE(STATUS "Will be building these examples: ${DD4HEP_EXAMPLES}")
```